### PR TITLE
Added delete_report function

### PR DESCRIPTION
**Pull request by ad08fee3**

### CODE REVIEW

File: dli_app/mod_reports/controllers.py [hunk 1]
```diff
@@ -117,3 +117,22 @@ def view_report(report_id):
...
+def delete_report(report_id):
+    """Delete the specified report"""
+    report = Report.query.get(report_id)
+    if report is None:
+        flash(
+            "Report not found!",
+            "alert-warning",
+        )
+    else:
+        db.session.delete(report)
```
Comment (gorel) [inline dli_app/mod_reports/controllers.py:138]: Recommendation: With this change, anyone can delete ANY report, even if it doesn't belong to him.
Add this check:

elif not current_user.is_admin and not report.user.id == current_user.id:
....flash that user does not have permission

We can handle the issue of reports being "favorited" later.

File: dli_app/mod_reports/controllers.py [hunk 2]
```diff
@@ -117,3 +117,27 @@ def view_report(report_id):
         return redirect(url_for('reports.my_reports'))
 
     return render_template('reports/view.html', report=report)
+
+
```
Comment (gorel) [inline dli_app/mod_reports/controllers.py:121]: One line between functions, then you can merge :)

**Comment by mozilla2012** [comment]
Uhhhhh look at the rest of the file bruh

File: dli_app/mod_reports/controllers.py (hunk 2 above)
Comment (ad08fee3) [inline dli_app/mod_reports/controllers.py:121]: Uhhhhh look at the rest of the file bruh

Comment (gorel) [inline dli_app/mod_reports/controllers.py:121]: Good job, you passed the test.

**Pull request merged**